ui/readme.md

```diff
@@ -1,39 +1,48 @@
 # PX Blue Colors
 
-This package contains PX Blue UI color definitions. These are intended for use as primary UI colors. 
+This package contains PX Blue UI color definitions. These are intended for use as primary UI colors.
 
 It contains the following color defintions:
+
 ```
 blue / $pxb-blue
-lightBlue / $pxb-lightBlue
-green / $pxb-green
+black / $pxb-black
+gray / $pxb-gray
+white / $pxb-white
+darkBlack / $pxb-darkBlack
```

Comment:
should this be vantaBlack?

I'm not opposed to aliasing vantaBlack with something a little less fancy like darkBlack though for those who don't know vanta black.

Reply:
I aliased it, but not sure if that will confuse people with smart IDE's auto completes

Reply:
I like PitchXtraBlack (PXB) too.  VantaBlack is scientifically-proven though (https://www.sciencealert.com/this-object-has-been-sprayed-with-the-world-s-blackest-pigment-and-it-s-freaking-us-out).

Reply:
I would suggest we make the default/documented color name darkBlack and let vantaBlack be an alias of that (just because I think people will more easily know what darkBlack is if they've never heard of vantaBlack. And our UI color palette doesn't have other colors with fun names (just lightBlue))